crates/aptos/src/move_tool/mod.rs

```diff
@@ -374,12 +436,20 @@ impl CliCommand<TransactionSummary> for PublishPackage {
                 .map(TransactionSummary::from)
         } else {
             // Send the compiled module and metadata using the code::publish_package_txn.
-            let metadata =
-                package.extract_metadata(upgrade_policy.unwrap_or_else(UpgradePolicy::compat))?;
+            let metadata = package.extract_metadata()?;
             let payload = aptos_transaction_builder::aptos_stdlib::code_publish_package_txn(
                 bcs::to_bytes(&metadata).expect("PackageMetadata has BCS"),
                 compiled_units,
             );
+            let size = bcs::serialized_size(&payload)?;
+            if !override_size_check && size > MAX_PUBLISH_PACKAGE_SIZE {
+                return Err(CliError::UnexpectedError(format!(
+                    "The package is larger than 60k ({}k)! To lower the size \
+                you may want to include less artifacts via `--included_artifacts`. \
+                You can also override this check with `--override-size-check",
+                    size / 1000
```

Comment:
I'd prefer if you just made both bytes so you could use the actual constant in the message.

Reply:
Using the constant now to construct the message.